internal/repositories/vehicles.go

```diff
@@ -137,31 +137,31 @@ func (v *Repository) GetVehicles(ctx context.Context, first *int, after *string,
 
 	if first != nil {
 		if last != nil {
-			return nil, errors.New("Pass `first` or `last`, but not both.")
+			return nil, errors.New("pass `first` or `last`, but not both")
 		}
 		if *first < 0 {
-			return nil, errors.New("The value for `first` cannot be negative.")
+			return nil, errors.New("the value for `first` cannot be negative")
 		}
 		if *first > maxPageSize {
-			return nil, fmt.Errorf("The value %d for `first` exceeds the limit %d.", *last, maxPageSize)
+			return nil, fmt.Errorf("the value %d for `first` exceeds the limit %d", *last, maxPageSize)
 		}
 		limit = *first
 	} else {
 		if last == nil {
-			return nil, errors.New("Provide `first` or `last`.")
+			return nil, errors.New("provide `first` or `last`")
 		}
 		if *last < 0 {
-			return nil, errors.New("The value for `last` cannot be negative.")
+			return nil, errors.New("the value for `last` cannot be negative")
 		}
 		if *last > maxPageSize {
-			return nil, fmt.Errorf("The value %d for `last` exceeds the limit %d.", *last, maxPageSize)
+			return nil, fmt.Errorf("the value %d for `last` exceeds the limit %d", *last, maxPageSize)
```

Comment:
We actually want this.

Reply:
Let me just figure out how to make VS Code ignore this. The problem is that we want nice sentences for the client but Go errors aren't supposed to have these. golangci-lint does not have a problem with it.